backend/src/apiserver/server/run_server_test.go

```diff
@@ -189,10 +200,16 @@ func TestCreateRun_Multiuser(t *testing.T) {
 	expectedRuntimeWorkflow.Labels = map[string]string{util.LabelKeyWorkflowRunId: "123e4567-e89b-12d3-a456-426655440000"}
 	expectedRuntimeWorkflow.Annotations = map[string]string{util.AnnotationKeyRunName: "run1"}
 	expectedRuntimeWorkflow.Spec.ServiceAccountName = "default-editor" // In multi-user mode, we use default service account.
+	template := expectedRuntimeWorkflow.Spec.Templates[0]
+	template.Metadata.Annotations = map[string]string{"sidecar.istio.io/inject": "false"}
+	template.Metadata.Labels = map[string]string{"pipelines.kubeflow.org/cache_enabled": "true"}
```

Comment:
Asking before I change too much again. In order to reuse the `addRuntimeMetadata` it needs to live outside of the test file(or be duplicated). What is best practice in this case? Should we just add `test_util.go` in order to make it exported?

Reply:
Agree, we can add a test_util.go